public/tenants/planet/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "Plant for the Planet",
+    "short_name": "Plant",
+    "start_url": "/?utm_source=pwa",
+    "description": "Plant trees around the world - Plant-for-the-Planet",
+    "categories": ["education"],
+    "display": "standalone",
```

Comment:
`standalone` or `fullscreen` looks best, but with rounded corners and notch, the back arrow for "tree registrations" or "profile editing" is not clickable. We have to decide with [display mode](https://developer.mozilla.org/en-US/docs/Web/Manifest/display) is best.